pipeline-scripts/release.groovy

```diff
@@ -116,6 +116,22 @@ Map stageValidation(String quay_url, String dest_release_tag, int advisory = 0,
         error("🚫 Advisory ${advisoryInfo.id} doesn't seem to be associated with a live ID.")
     }
 
+    if (arch == 'amd64' || arch == 'x86_64') {
```

Comment:
All arches should be validated.

Reply:
well; yes. note, though, that some images don't build in all arches, and missing entries are replaced in the release image by dummy images (from the `cli` image) so non-x86 are likely to be "missing" some of the contents that are attached to the advisory. `elliott verify-payload` needs to get a little smarter to handle those.

Reply:
although i tried it out last time i was promoting stuff and it did not flag anything about the non-x86 arches. maybe you already made it smart enough?